tests/unit/api/bucketDelete.js

```diff
@@ -14,7 +14,7 @@ const canonicalID = 'accessKey1';
 const authInfo = makeAuthInfo(canonicalID);
 const namespace = 'default';
 const bucketName = 'bucketname';
-const postBody = new Buffer('I am a body');
+const postBody = Buffer.from('I am a body', 'binary');
```

Comment:
hmm, that means only cryptographic encoding is binary by default, will change that :)